### ISSUE TITLE
Class not found

### ISSUE THREAD

**Issue by katmaha** [drewm/mailchimp-api#56]
Hi, I'm using a local WAMP server, php 5.3. I enabled curl even though it said not necessary.
I did not use Composer, just include MailChimp.php, and I get this error:

Fatal error: Class 'MailChimp' not found in ...

I do an echo in the file to make sure it's being included and it is. If I comment out the namespace line I don't get the error anymore but nothing happens either. I mean my form contact form gets posted, does other backend stuff like add form details to my db but that's it.
I'm using the add subscriber example.
help!

**Comment by katmaha** [comment]
Ok perhaps I'm a noob but I figured it out. I guess I haven't used php namespaces before, didn't realise I had to include the namespace here: 
MailChimp = new \DrewM\MailChimp\MailChimp(apikey);

Don't get the error anymore but still not adding a subscriber to my list.
If I use api v2 wrapper it works fine.

This is my code:
$MailChimp = new \DrewM\MailChimp\MailChimp($myapi);
$result = $MailChimp->post('lists/listid/members', array(
                'email_address'     => $email,
                'status'            => 'subscribed',
                'merge_fields'      => array('FNAME'=>$fname)
));
print_r($result);

So with v2 I get the result array printed out and address added to the list, but not with v3 - neither the printed result or added subscriber. Any help?
